tests/test_graph.py

```diff
@@ -2,6 +2,7 @@
 import unittest
 from openeo_pg_parser.translate import translate_process_graph
 
+OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/processes.json"
```

Comment:
To ensure that tests are not failing due to a new release, I'd propose to link against a specific version.

```suggestion
OPENEO_PROCESSES_ENDPOINT = "https://processes.openeo.org/1.2.0/processes.json"
```